### ISSUE TITLE
PyPi installation does not install cached-property

### ISSUE THREAD

**Issue by smajee** [tomography/xdesign#77]
Even though `cached-property` appears in the `requirements.txt`, installing `xdesign` from PyPi does not install `cached-property`.

Could it be due to `cached-property` not being listed in `install_requires` in `setup.py`?

**Comment by bwohlberg** [comment]
This needs to be confirmed, but I suspect that this issue was resolved by #78.

**Comment by carterbox** [comment]
It's not resolved until the next release is published.

**Comment by bwohlberg** [comment]
Can it be considered resolved with the release of 0.6.0?

**Comment by carterbox** [comment]
It's not resolved until the release on 0.6.0 *on PYPI*.

**Comment by bwohlberg** [comment]
Any planned timeline for that?

**Comment by carterbox** [comment]
No. I don't control the account which publishes this package on PYPI.

**Comment by bwohlberg** [comment]
@dgursoy: As the owner of the PyPI package, could you please update it?
